Modules/Filtering/ImageCompose/test/itkComposeBigVectorImageFilterTest.cxx

```diff
@@ -58,22 +58,23 @@ itkComposeBigVectorImageFilterTest(int, char *[])
   VectorImageType::Pointer img = composeFilter->GetOutput();
   std::cout << "Compose filter executed." << std::endl;
 
+
+  itk::ImageRegion<3> sliceRegion = img->GetLargestPossibleRegion();
+  sliceRegion.SetSize(2, 1); // Set the size of the z-dimension to 1
+
   for (unsigned int i = 0; i < nchannels; ++i)
   {
     itk::IndexValueType                  z = i;
     std::map<unsigned int, unsigned int> values;
 
-    for (itk::IndexValueType y = 0; y < size; ++y)
+    sliceRegion.SetIndex(2, z); // Set the index of the z-dimension to the current slice
+
+    itk::ImageRegionConstIterator<VectorImageType> it(img, sliceRegion);
+    for (it.GoToBegin(); !it.IsAtEnd(); ++it)
```

Comment:
@blowekamp Cool. 👍  Some small suggestions...

`it.GoToBegin()` is not necessary. An `ImageRegionConstIterator` always starts at the begin. So these two lines could be replaced with one:

```cpp
    for (itk::ImageRegionConstIterator<VectorImageType> it(img, sliceRegion); !it.IsAtEnd(); ++it)
```